tests/trestlebot/cli/test_sync_cac_content_cmd.py

```diff
@@ -176,3 +182,144 @@ def test_sync_product_create_validation_component(tmp_repo: Tuple[str, Repo]) ->
     assert len(component.props) == 30
     assert component.title == "openscap"
     assert component.type == "validation"
+
+
+def test_missing_required_profile_option(tmp_repo: Tuple[str, Repo]) -> None:
+    """Tests missing required option in sync-cac-content-profile command."""
+
+    repo_dir, _ = tmp_repo
+    repo_path = pathlib.Path(repo_dir)
+
+    setup_for_catalog(repo_path, test_cat, "catalog")
+
+    runner = CliRunner()
+    result = runner.invoke(
+        sync_cac_content_profile_cmd,
+        [
+            "--cac-content-root",
+            str(test_content_dir),
+            "--product",
+            test_product,
+            "--oscal-catalog",
+            test_cat,
+            "--repo-path",
+            str(repo_path.resolve()),
+            "--committer-email",
+            "test@email.com",
+            "--committer-name",
+            "test name",
+            "--branch",
+            "test",
+        ],
+    )
+    assert result.exit_code == 2
+
+
+def test_profile_supplied(tmp_repo: Tuple[str, Repo]) -> None:
```

Comment:
I think this case could be merged to the next case: test_created_oscal_profile. The two cases have the same check point of creating the oscal profile successfully. What do you think?